### DISCUSSION TITLE
I don't remember the password...

### DISCUSSION THREAD

**Discussion by FenixNew** [redeclipse/base#1471] [Help and Support] [answered]
Good afternoon, I want to change the password of my game account, please can someone from the administration help me!

**Comment by dc-redeclipse** [comment]
Hello FenixNew,

game Account? Are you talking about your Red Eclipse account, do you mean your Auth key with the password you like to change?

Besides the Red Eclipse account used in the past, we don't feature any other kinds of accounts within the game.

Auth keys can't be changed and should be kept secret at any time!

If you are talking about your Steam account as a game account, you can change it in Steam following this: [How to change your Steam Password](https://help.steampowered.com/en/faqs/view/5107-700D-89B4-A4CD)
If you lost your password, follow this: [How to Recover Steam Password](https://help.steampowered.com/en/wizard/HelpWithLoginInfo?issueid=406)

Hope that helps.

Greetings
dc

**Comment by FenixNew** [comment]
Good evening, I'm talking about my Red Eclipse game account, inside the game there is the option of "user account", which you log in with email and password.

**Reply by dc-redeclipse** [reply]
Oh wow, then you are talking about a thing from an old Red Eclipse version back in IRC times. Which version you are using?

**Reply by FenixNew** [reply]
Red Eclipse is:v1.6.0-nix64 sunset edition

**Comment by FenixNew** [comment]
Good evening, I'm talking about my Red Eclipse game account, inside the game there is the option of "user account", which you log in with email and password.

**Comment by FenixNew** [comment]


**Reply by dc-redeclipse** [reply]
Which Red Eclipse version are you using?
**Please don't post private informations like your email address here or anywhere else, or give it to anyone! Never do that!**

**Reply by FenixNew** [reply]
Red Eclipse is:v1.6.0-nix64 sunset edition

**Comment by qreeves** [comment]
I've deleted the private information.

Red Eclipse accounts have been discontinued and I am only providing limited support to existing users who are not moderators.

Please DM me on [Discord](https://www.redeclipse.net/discord) so I can sort out a new auth key. I am `@qreeves`.

**Comment by FenixNew** [comment]
The version of Red Eclipse is:v1.6.0-nix64 sunset edition

**Comment by FenixNew** [comment]
Please, I'm waiting for answers!

**Reply by dc-redeclipse** [reply]
Please DM Quin on Discord as he already told you. Join our Discord agree to the rules and write him.

**Comment by FenixNew** [comment]
Good evening guys, I'm waiting for the password...

**Reply by dc-redeclipse** [reply]
If you don't want to follow the instructions and ways to get help, this will never happen, and you will wait forever.
You have still not joined our [Discord](https://www.redeclipse.net/discord), so no way to get in touch via DMs before that happens.

**Comment by FenixNew** [comment, marked as answer]
Good evening guys, DM Quin has already solved it, I'm very grateful for everyone trying to help me...
